README.md

```diff
@@ -34,8 +45,24 @@ $ npm run dev -- --env
 
 To get an email from gov notify, you'll need to be added to the email list for testing on [gov notify](https://www.notifications.service.gov.uk/sign-in)
 
-## Skip email verify step
+## Skip email verify step  <a name="skip-email-verify-step"></a>
 
 On dev, preprod and locally you can skip the email verify step by going to the following url
 
 http://localhost:8081/nrm/start?token=skip
+
+
+## Acceptance Tests  <a name="acceptance-tests"></a>
+
+[Google Puppeteer](https://developers.google.com/web/tools/puppeteer/) is used to run our end-to end test
+scripts for the application. It is a Node library which provides a high-level API to control headless Chrome or Chromium over the DevTools Protocol. It can also be configured to use full (non-headless) Chrome or Chromium.
+
+### Running Acceptance Tests  <a name="running-acceptance-tests"></a>
+
+[Install & Run](#install-and-run)  the application then you can run the acceptance tests using the command below:
+
+```bash
+$ npm run start:local-acceptance
```

Comment:
the above didn't work for me ☝️ are you sure it's not `npm run local-acceptance`

Reply:
I have updated the README.md, the new commands are 
test:docker-acceptance -  test within a docker container
test:local-acceptance - test locally
test:local-acceptance-demo - test locally but in demo mode (shows browser and runs much slower)